docs/src/_parts/template-tutorial

```diff
@@ -0,0 +1,97 @@
+# Tutorial docs template
+
+> Tutorials are explained guides which fulfil the objectives of learning and
```

Comment:
```suggestion
> Tutorials are explained guides which fulfill the objectives of learning and
```

Reply:
You really need to turn that US dictionary off :)

Reply:
Nope. You really need to turn that US dictionary off ;)